config/locales/es.yml

```diff
@@ -3029,19 +3029,19 @@ es:
         edit:
           affirmative: Sí, me otorgaron una extensión federal
           az:
-            affirmative: Sí, me otorgaron una extensión federal
-            negative: No, no me otorgaron una extensión federal
-            subtitle_2: 'Te habrían concedido una extensión federal si realizaste alguna de las siguientes acciones en o antes del 15 de abril:'
-            title: "¿Te otorgaron una extensión federal?"
+            affirmative: Sí, presenté una solicitud para una extensión federal
+            negative: No, no presenté una solicitud para una extensión federal
+            subtitle_2: 'Presentaste una solicitud para una extensión federal si realizaste alguna de las siguientes acciones en o antes del 15 de abril:'
+            title: "¿Presentaste una solicitud para una extensión federal?"
           list_1: Hiciste un pago de extensión federal
           list_2_html: Presentaste el <a href="https://www.irs.gov/pub/irs-pdf/f4868.pdf" target="_blank" rel="noopener noreferrer">Formulario 4868</a> en línea o lo enviaste por correo postal
           negative: No, no me otorgaron una extensión federal
           nj:
-            affirmative: Sí, me otorgaron una extensión federal
-            negative: No, no me otorgaron una extensión federal
-            subtitle_2: 'Te habrían concedido una extensión federal si realizaste alguna de las siguientes acciones en o antes del 15 de abril:'
-            title: "¿Te otorgaron una extensión federal?"
-          subtitle_1: Si te aceptaron una solicitud de extensión federal, esto significa que puedes presentar tus declaraciones de impuestos federales y estatales hasta el 15 de octubre sin incurrir en una multa por no presentar.
+            affirmative: Sí, presenté una solicitud para una extensión federal
+            negative: No, no presenté una solicitud para una extensión federal
+            subtitle_2: 'Presentaste una solicitud para una extensión federal si realizaste alguna de las siguientes acciones en o antes del 15 de abril:'
+            title: "¿Presentaste una solicitud para una extensión federal?"
+          subtitle_1: Si presentaste una solicitud para una extensión federal, esto significa que puedes presentar tus declaraciones de impuestos federales y estatales hasta el 15 de octubre sin incurrir en una multa por no presentar.
```

Comment:
`subtitle_1` is shared across all states, does this spanish copy apply to NC as well?

Also for context, I'd love to refactor this at some point. Only NC, AZ, NJ use this page & so far NC copy is treated as the "default" although it's technically in the minority. This is why az/nj have their own yaml blocks in this file, it'd be easier if we had default as the az/nj copy & have a `nc:` block for its specific copy
With that said, it probably doesn't matter too much unless we decide to use this controller for more states.